aws/resource_aws_sqs_queue_policy.go

```diff
@@ -88,8 +90,23 @@ func resourceAwsSqsQueuePolicyUpsert(d *schema.ResourceData, meta interface{}) e
 		}
 		return nil
 	})
+	if isResourceTimeoutError(err) {
+		out, err = conn.GetQueueAttributes(gqaInput)
+		if err == nil {
+			queuePolicy, ok := out.Attributes[sqs.QueueAttributeNamePolicy]
+			if !ok {
+				return fmt.Errorf("SQS queue attribute not found")
```

Comment:
Seems like we should reuse the `notUpdatedError` above for consistency here 👍 

```suggestion
				return notUpdatedError
```